substrate/runtime-support/src/dispatch.rs

```diff
@@ -47,6 +46,9 @@ pub trait AuxCallable {
 	type Call: AuxDispatchable + Slicable + Clone + PartialEq + Eq;
 }
 
+// dirty hack to work around serde_derive issue.
```

Comment:
Would be good to know more about this issue - a link & description of what to do when the issue is sorted.

Reply:
https://github.com/rust-lang/rust/issues/51331

see the linked serde issue